scrapers/scrape_bs.py

```diff
@@ -130,11 +130,14 @@
     sc.find(r'Todesfälle\s*im\s*Kanton\s*Basel-Stadt\s*beträgt(?:\s*\S+)?\s*insgesamt\s*([0-9]+)\b', d) or \
     sc.find(r'Die\s*Zahl\s*der\s*Todesfälle\s*im\s*Kanton\s*Basel-Stadt\s*beträgt\s*.*unverändert\s*([0-9]+)\b', d)
 
-isolated = sc.int_or_word(sc.find(r'\s+(\S+)\s+aktiven\s+Fällen', d))
+isolated = sc.int_or_word(sc.find(r'\s+(\S+)\s+aktiven\s+(?:Fällen|Fall)', d))
 if dd.hospitalized is not None and isolated is not None:
     isolated = int(isolated) - int(dd.hospitalized)
 dd.isolated = isolated
-dd.quarantined = sc.int_or_word(sc.find(r'In\s+Quarantäne\s+befinden\s+sich\s+(?:aktuell\s+)?(\S+)\s+Personen', d))
+if 'In Quarantäne befindet sich seit gestern Montag niemand mehr.' in d:
```

Comment:
Maybe this could be a regex something like

```python
if re.search(r'In Quarantäne befindet sich .*? niemand mehr', d):
```

since I expect this to be a back-and-forth thing.

Reply:
Good point. Updated

Reply:
You didn't yet push the updated code. Other than that, this is ready to be merged.

Reply:
Done now. Sorry about that